packages/dd-trace/test/appsec/sdk/index.spec.js

```diff
@@ -1,200 +0,0 @@
-'use strict'
```

Comment:
Is this delete intentional?

Reply:
nope, it is an unexpected result from the rebasing. I've restored it.